frontend/benefit/handler/src/components/applicationList/ApplicationList.tsx

```diff
@@ -96,22 +96,22 @@ const ApplicationList: React.FC<ApplicationListProps> = ({
         Cell: ({
           cell: {
             row: {
-              original: { additionalInformationNeededBy },
+              original: { additionalInformationNeededBy, status: itemStatus },
             },
           },
         }) => (
-          <$DataValueAlert
-            css={`
-              font-weight: 400;
-            `}
-          >
-            {t(
-              `common:applications.list.columns.additionalInformationNeededByVal`,
-              {
-                date: convertToUIDateFormat(additionalInformationNeededBy),
-              }
-            )}
-          </$DataValueAlert>
+          <div>
+            {itemStatus === APPLICATION_STATUSES.INFO_REQUIRED ? (
```

Comment:
unneeded check? already checked on line 93? Or what's the difference between status and itemStatus

Reply:
yes, as we have applications with multiple statuses in the list